torch/csrc/jit/script/compiler.cpp

```diff
@@ -2035,6 +2085,70 @@ struct to_ir {
     } else {
       auto inputs = getNamedValues(apply.inputs(), true);
       auto attributes = emitAttributes(apply.attributes());
+
+      auto r = getFullyQualifiedNameAndSchema(apply.callee(), inputs, attributes);
```

Comment:
I do not think this is the right approach. It replicates a bunch of the SugaredValue behavior locally just to try to get the fully-qualified name for a function relative to the module root. MethodValue should just track its fully qualified name itself. I would not expect to see many changes in compiler.cpp